source/query-with-natural-language/prompt-natural-language-query.txt

```diff
@@ -9,6 +9,99 @@ Prompt your First Natural Language Query
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 2
+   :depth: 1
 
-Prompt your First Natural Language Query
\ No newline at end of file
+You can use MongoDB Compass to generate filter documents using the
+natural language querying feature. This feature automatically generates 
+filter documents based on prompts you provide to the MongoDB AI engine. 
+Natural language querying serves as a helpful starting point and assists
+you in writing MongoDB queries.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-natural-language-query.rst
+
+The MongoDB AI engine can return both filter documents and 
+aggregations. If your prompt results in an aggregation, a popup
+appears directing you to the :guilabel:`Aggregations` tab.
+
+This page uses the :ref:`sample_mflix.movies <sample-mflix>` 
+collection from the Atlas sample dataset.
+
+Before you Begin
+----------------
+
+You must :ref:`enable the natural language querying feature<enable-natural-language-querying>`.
+
+Steps
+-----
+
+.. procedure::
+   :style: connected
+
+   .. step:: Select the :guilabel:`Documents` tab
+
+   .. step:: Click the :guilabel:`Ask AI` button
+
+      The :guilabel:`Natural Language Query Bar` appears.
+
+   .. step:: Type a natural language prompt for your collection
+
+      Enter a prompt in the :guilabel:`Natural Language Query Bar`.
+      For example: ``Which movies were released in 2000?``.
+
+   .. step:: Press enter or click :guilabel:`Generate`
+
+      A filter document populates in the :guilabel:`Filter` bar.
+
+   .. step:: Carefully examine the filter document
+
+      Before running the query, make sure to thoroughly 
+      review the syntax.
+
+   .. step:: Click the :guilabel:`Find` button to execute the query
+
+      The results populate in the documents view.
+
+Example
+-------
+
+Below are examples of prompts to help you understand expected results 
+when using Natural Language Querying. For more examples and best practices, 
+see :ref:`natural-language-best-practices`.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
+
+   * - Prompt
+     - Response
+
+   * - ``Which movies have a "PG" rating?``
+     - .. code-block:: json
+          :copyable: false
+
+          {"rated": "PG"}
+
+   * - ``Which movies include a writer of "David Mamet"?``
```

Comment:
(suggestion)
I might say something like "Which movies were written by David Mamet?" instead

Reply:
Well shoot. Then this is a very good example. Thanks for explaining. What do you think about adding a similar explanation to the example?

Reply:
I have rephrased the question to make this more obvious:
`Which movies include "David Mamet" in the writer array field?`